### PR TITLE
Adding note to delta download failure log

### PR DESCRIPTION
**Pull request by rmorillo24**
Change-type: patch
Signed-off-by: Rafael Morillo <rafael@balena.io>

### CODE REVIEW

**Comment by Hades32** [comment]
@rmorillo24 the `Change-type: patch` must be in the commit message, not (only) in the PR message. The easiest way to fix this is to do a `git commit --amend`, add the patch type to the message and then a `git push --force`

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

File: pages/learn/deploy/delta.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ Before devices can update using deltas, a delta must be generated between the re
 
 On-demand deltas may take a while to generate, depending primarily on the size of the images involved. While the delta is generating, the device keeps polling the API, checking if the delta is ready. If a delta already exists, this step is skipped.
 
+if the delta is being generated, and the supervisor tries to download it, you will get this line in your supervisor log file:
+
+'''
+Failed to download application 'registry2.resin.io/foo/bar' due to 'Delta server is still processing the delta, will retry
```
Comment (dfunckt) [inline pages/learn/deploy/delta.md:32]: > Failed to download application 'registry2.resin.io/foo/bar' due to 'Delta server is still processing the delta, will retry

Are you sure logs print this message?

**Review by dfunckt — CHANGES_REQUESTED** [review]

File: pages/learn/deploy/delta.md (hunk 1 above)
Comment (rmorillo24) [reply pages/learn/deploy/delta.md:32]: @dfunckt , I was following the issue attached together with some comment in this Front thread https://app.frontapp.com/open/cnv_l1pu4x
But you are right, I'm not sure if this is really printed, and if it is if this is the cause.

**Comment by vipulgupta2048** [comment]
Closing, please do re-open if they are more changes to be made.

**Pull request closed without merging**